bootstrap/utils.go

```
@@ -29,13 +29,13 @@ func downloadFile(url string, path string) error {
 	if err != nil {
 		return err
 	}
-	defer output.Close()
+	defer output.Close() // nolint: errcheck
```

we need to comment here because we're ignoring the error